src/Tizen.Multimedia.Remoting/MediaController/MediaControlPlaylist.cs

```diff
@@ -0,0 +1,120 @@
+            }
+
+            Name = name ?? throw new ArgumentNullException("Playlist name is not set.");
+            Position = position;
+        }
+
+        /// <summary>
+        /// Gets or sets the index of the media in the playist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public string Index { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets the name of playist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public string Name { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets the playing position of the media indicated by <see cref="Index"/>.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public ulong Position { get; protected set; }
+
+        /// <summary>
+        /// Gets the total number of media in the playlist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public virtual int TotalCount { get; } = 1;
+
+        /// <summary>
+        /// Creates an object of the MediaControlPlaylist with the specified name, index, position.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="index"></param>
+        /// <param name="position"></param>
+        /// <returns>A <see cref="MediaControlSimplePlaylist"/> instance.</returns>
+        /// <exception cref="ArgumentException"><paramref name="index"/> cannot be converted to number.</exception>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="name"/> or <paramref name="index"/> is not set.
+        /// </exception>
+        /// <since_tizen> 5 </since_tizen>
+        public static MediaControlSimplePlaylist Create(string name, string index, ulong position)
+        {
+            return new MediaControlSimplePlaylist(name, index, position);
+        }
+
+        /// <summary>
+        /// Creates an object of the MediaControlPlaylist with the specified name, index.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="index"></param>
+        /// <returns>A <see cref="MediaControlSimplePlaylist"/> instance.</returns>
+        /// <exception cref="ArgumentException"><paramref name="index"/> cannot be converted to number.</exception>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="name"/> or <paramref name="index"/> is not set.
+        /// </exception>
+        /// <since_tizen> 5 </since_tizen>
+        public static MediaControlSimplePlaylist Create(string name, string index)
+        {
+            return new MediaControlSimplePlaylist(name, index, 0);
+        }
+    }
+
+
+    /// <summary>
+    /// Represents simple playlist for media control.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public class MediaControlSimplePlaylist : Playlist
+    {
+        internal MediaControlSimplePlaylist(string name, string index, ulong position)
```

Comment:
Why do you disallow call the constructor by user?

Reply:
Even if you provide a factory method, I don't think there is no reason to make internal constructor